integration-tests/aws2/src/main/java/org/apache/camel/quarkus/component/aws/CamelRoute.java

```diff
@@ -14,21 +14,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.camel.quarkus.component.aws2.sqs.it;
+package org.apache.camel.quarkus.component.aws;
 
-import io.quarkus.test.junit.QuarkusTest;
-import io.restassured.RestAssured;
-import org.junit.jupiter.api.Test;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+import org.apache.camel.builder.RouteBuilder;
 
-@QuarkusTest
-class Aws2SqsTest {
+@RegisterForReflection
```

Comment:
I do not think `@RegisterForReflection` is necessary here

Reply:
No, it's not needed